window/UserControlHost.vb

```diff
@@ -0,0 +1,93 @@
+﻿Imports System.Runtime.InteropServices
+Imports System.Windows.Forms
+Imports System.Drawing
+
+<ComVisible(True), Guid("0EB93108-D229-4F6F-82C5-0B96AFFBB9C5"), ProgId("MyVBAAddin.UserControlHost")>
+Public Class UserControlHost

```

Comment:
Please add brief comments about what this class does
What is GUID.
Please change the name of add in

Reply:
Add a TODO to change the name of the add in.

Reply:
Done.